gc-metrics.js

```diff
@@ -17,14 +17,16 @@ const gcTypes = {
 
 const noop = () => {};
 
-export default () => {
+export default client => {
+  const promClient = client || { Counter };
```

Comment:
No worries! Don't hesitate to update the PR early if you want feedback as you go. Just make sure to leave a note in the issue and not just push, so that I get an email 😄